src/index.js

```diff
@@ -284,17 +284,20 @@ class ClientJS extends NxusModule {
         this.log.debug('Browserify bundle for', entry, msg)
       })
     let bundle = () => {
-      b.bundle()
-        .on('error', (err) => {
-          this.log.error('Bundle error for', entry, err)
+      return new Promise((resolve, reject) => {
+        b.bundle((err, buf) => {
+          if (err) {
+            this.log.error('Bundle error for', entry, err)
+            reject(err)
+            return
+          }
+          fs.writeFileSync(outputFile, buf)
+          resolve()
         })
-        .on('log', (msg) => {
```

Comment:
I do, sometimes, appreciate the 'it worked' log message here when debugging - otherwise this change looks good to me.